src/backend/aspen/api/schemas/samples.py

```diff
@@ -1,12 +1,45 @@
+from typing import List, Optional
+
 from pydantic import constr
 
-from aspen.api.schemas.base import BaseRequest
+from aspen.api.schemas.base import BaseRequest, BaseResponse
 
 
 class SampleRequestSchema(BaseRequest):
     # mypy + pydantic is a work in progress: https://github.com/samuelcolvin/pydantic/issues/156
     name: constr(min_length=1, max_length=128, strict=True)  # type: ignore
 
 
-class SampleResponseSchema(BaseRequest):
-    name: str
+class SampleGisaidResponseSchema(BaseResponse):
+    gisaid_id: Optional[str]
+    status: str
+
+
+class SampleLineageResponseSchema(BaseResponse):
+    last_updated: Optional[str]
+    lineage: Optional[str]
+    probability: Optional[str]
+    version: Optional[str]
+
+
+class SampleGroupResponseSchema(BaseResponse):
+    group_id: int
+    group_name: str
```

Comment:
This is already under a "submitting_group" key. I think these should just be "id" and "name"